packages/perseus/src/widgets/table/score-table.ts

```diff
@@ -14,25 +17,13 @@ function scoreTable(
     rubric: PerseusTableRubric,
     strings: PerseusStrings,
 ): PerseusScore {
-    const filterNonEmpty = function (table: any) {
```

Comment:
This function moved to `utils.ts` because it's needed by both the scoring and the new validation behaviour.